Solution-Week1/script_5_count_vowels.py

```diff
@@ -0,0 +1,10 @@
+#Write A Script To Enter Any String And Count Vowels
+def count_vowels(string):
+    a=0
+    vowels=['a','e','i','o','u','A','E','I','O','U']
+    for i in string:
+        if i in vowels:

```

Comment:
any other way to write this statement? (without using in).

Reply:
Yes Sir, i can also write this statement using == operator.
 
def count_vowels(string):
    a=0
    vowels=['a','e','i','o','u','A','E','I','O','U']
    for i in string:
        for j in vowels:
            if i == j:
                a+=1
    print(f"Vowels In String Are {a}")
string=input("\n Which String You'r Gonna Enter ...? :")
count_vowels(string)